webserver/webserver/src/main/java/io/helidon/webserver/BareResponseImpl.java

```diff
@@ -78,19 +78,19 @@
     private volatile DefaultHttpResponse response;
     private volatile boolean lengthOptimization;
     private volatile boolean isWebSocketUpgrade = false;
-    private CompletableFuture<?> prev;
+    private CompletableFuture<?> previousRequestFuture;
```

Comment:
Since we are into renaming, I'd like to make sure the meaning is documented to be not only previous *request*, but really any chunk.

(N.B.: `onNext` may push several content chunks for the same request, so `prev` would reference the future for those chunks)